packages/@react-spectrum/avatar/test/Avatar.test.js

```diff
@@ -31,26 +30,6 @@ describe('Avatar', () => {
         width: '80px'
       });
     });
-
-    // Spying on dimensionValue since we're unable to use toHaveStyle effectively with CSS vars
-    // See https://github.com/testing-library/jest-dom/issues/322
-    it('supports predefined avatar sizes', () => {
-      const dimensionValueSpy = jest.spyOn(Utils, 'dimensionValue');
-      render(<Avatar src="http://localhost/some_image.png" size="avatar-size-700" />);
-      expect(dimensionValueSpy).not.toHaveBeenCalledWith('avatar-size-100');
-    });
-
-    it('defaults to default size when size is size-XXXX', () => {
-      const dimensionValueSpy = jest.spyOn(Utils, 'dimensionValue');
-      render(<Avatar src="http://localhost/some_image.png" size="size-100" />);
-      expect(dimensionValueSpy).toHaveBeenCalledWith('avatar-size-100');
-    });
-
-    it('defaults to default size when size is a string number', () => {
-      const dimensionValueSpy = jest.spyOn(Utils, 'dimensionValue');
-      render(<Avatar src="http://localhost/some_image.png" size="100" />);
-      expect(dimensionValueSpy).toHaveBeenCalledWith('avatar-size-100');
-    });
```

Comment:
These tests didn't seem particularly useful now that we have chromatic. Could be fixed by switching to module mocks like I did in focus tests but didn't seem worth it.